__tests__/app.test.js

```diff
@@ -129,4 +129,103 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+  describe("PATCH", () => {
+    test("Status:201, the body should contain a property of inc_votes and should have a number as a value", () => {
+      const addVotes = { inc_votes: -100 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(addVotes)
+        .expect(201)
+        .then(({ body }) => {
+          expect(addVotes).toHaveProperty("inc_votes");
+          expect(addVotes.inc_votes).toEqual(expect.any(Number));
+        });
+    });
+    test("Status:201, should update votes for the requested article_id", () => {
+      const addVotes = { inc_votes: 100 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(addVotes)
+        .expect(201)
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
+            expect(article).toEqual(expect.objectContaining({ article_id: 1 })),
+              expect(article.article_id).toEqual(1);
+          });
+        });
+    });
+    test("Status:201, should add the votes when given positive number", () => {
+      const addVotes = { inc_votes: 100 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(addVotes)
+        .expect(201)
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("votes"),
+              expect(article.votes).toEqual(200);
+          });
+        });
+    });
+    test("Status:201, should subtract the votes when given negitive number", () => {
+      const addVotes = { inc_votes: -50 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(addVotes)
+        .expect(201)
+        .then(({ body: { articles } }) => {
+          articles.forEach((article) => {
+            expect(article.votes).toEqual(50);
+          });
+        });
+    });
+    test("Status:201, should not change any other properties", () => {
+      const addVotes = { inc_votes: -50 };
```

Comment:
This test also cannot fail, you are using a patch request to change data in the data base, the originalArticle object is never changed. Line 203 and 204 are not comparing to it so this test is passing. This is not an appropriate mutation test